src/components/Dashboard/WithDataDisclaimer/components/DataDisclaimer/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { Button, Col, Container, Form, Modal } from "react-bootstrap";
+import { BOOLEAN_ENUM } from "../../../../../constants";
```

Comment:
just as a suggestion, something that can be done in the future is to add a path alias to make this more readable

Reply:
Great suggestion, thank you so so much, appreciate it.

Reply:
I'll add it in another ticket https://github.com/rivasvict/react-expenses-manager/issues/66